src/plugin.js

```diff
@@ -237,6 +237,20 @@ const onPlayerReady = (player) => {
 const eme = function(options = {}) {
   this.eme.options = options;
 
+  this.eme.init = (config) => {
```

Comment:
We may want to consider a more precise name. Maybe something as direct as `initializeMediaKeys`, and provide a comment/docs around why you'd want to use this API.